examples/lib/defaultHtmlSequencerUi.js

```diff
@@ -14,12 +14,13 @@ function DefaultHtmlSequencerUi(_sequencer, options) {
     var hash = getUrlHashParameter("steps");
 
     if (hash) {
-      var stepsFromHash = hash.split(",");
-      stepsFromHash.forEach(function eachStep(step) {
-        _sequencer.addSteps(step);
+      var stepsFromHash = _sequencer.toJson(hash);
+      stepsFromHash.forEach(function eachStep(stepObj) {
+        _sequencer.addSteps(stepObj.name,stepObj.options);
```

Comment:
great, this would eventually be replaced by `importString` or what were we going to call it?

Reply:
@jywarren yeah We can call it importString()

Reply:
@jywarren actually I split on '),' !!

Reply:
Hmm, I wonder if this could still be triggered by incorrect parsing with a specifically crafted function. 

Maybe we should forbid functions unless it's a full json string? Or could we url-encode the function like with `%20`-like encoding? But that would be quite unreadable.

Reply:
@jywarren this won't fail until the function is incorrect!!

Reply:
In which case even if we encode it won't make sense since it would not be valid